src/settings.ts

```diff
@@ -81,6 +83,7 @@ export interface Settings {
   beforeCurrent?: Actions;
   afterCurrent?: Actions;
   blankMigrationContent?: string;
+  logFactory?: MigrateLogFactory;
```

Comment:
Let's accept a pre-built Logger instead of a logFactory; this is inline with how we do it in Worker: https://github.com/graphile/worker#logger

```suggestion
  logger?: Logger;
```

Reply:
I'm fine with this change, as long as it does not force consumer of `@graphile/migrate` to explicitly install `@graphile/logger` :)
(Added an extra Logger re-export to support this)